stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -134,20 +134,29 @@ public struct Unsafe${Mutable}RawBufferPointer
   }
 
 %  if mutable:
+  @available(*, unavailable, renamed: "allocate(byteCount:alignment:)")
+  public static func allocate(count: Int) -> UnsafeMutableRawBufferPointer { 
+    fatalError("unreachable") 
+  }
+
   /// Returns a newly allocated buffer with the given size, in bytes.
   ///
   /// The memory referenced by the new buffer is allocated, but not
   /// initialized.
   ///
-  /// - Parameter size: The number of bytes to allocate.
-  /// - Returns: A word-aligned buffer pointer covering a region of memory.
+  /// - Parameters:
+  ///   - byteCount: The number of bytes to allocate.
+  ///   - alignment: The alignment of the new region of allocated memory, in
+  ///     bytes.
+  /// - Returns: A buffer pointer to a newly allocated region of memory aligned 
+  ///     to `alignment`.
   @_inlineable
-  public static func allocate(count size: Int
-  ) -> UnsafeMutableRawBufferPointer {
-    return UnsafeMutableRawBufferPointer(
-      start: UnsafeMutableRawPointer.allocate(
-        bytes: size, alignedTo: MemoryLayout<UInt>.alignment),
-      count: size)
+  public static func allocate(
+    byteCount:Int, alignment:Int
+    ) -> UnsafeMutableRawBufferPointer {
```

Comment:
Nit: space after colons in `byteCount:` and `alignment:`, then deindent the closing parenthesis:

```swift
public static func allocate(
  byteCount: Int, alignment: Int
) -> UnsafeMutableRawBufferPointer {
  // ...
}
```